miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -50,7 +50,12 @@
 //   * mesh-optimizer -m square01.mesh -o 2 -rs 2 -mid 87 -tid 8 -ni 10  -ls 2 -li 100 -bnd -qt 1 -qo 8 -fd 1 -ae 1
 //   Adapted discrete aspect ratio (3D)
 //     mesh-optimizer -m cube.mesh -o 2 -rs 2 -mid 302 -tid 7 -ni 20  -ls 2 -li 100 -bnd -qt 1 -qo 8
-
+//
+//   Adaptive limiting:
+//     mesh-optimizer -m stretched2D.mesh -o 2 -mid 2 -tid 1 -ni 50 -qo 5 -nor -vl 1 -alc 0.5 -ae 0
+//   Adaptive limiting through FD (requires GSLIB):
+//     *  mesh-optimizer -m stretched2D.mesh -o 2 -mid 2 -tid 1 -ni 50 -qo 5 -nor -vl 1 -alc 0.5 -fd -ae 1
```

Comment:
To suppress this from being tested, you need to have one space after the `*`, not two.